docs/configuration/json-configuration-file.md

```diff
@@ -46,6 +48,7 @@ Here are the rules:
    - `bssid` and `channel` have to be defined together and these settings are independand of settings related to static IP
    - to define static IP, `ip` (IP address), `mask` (netmask) and `gw` (gateway) settings have to be defined at the same time
    - to define second DNS `dns2` the first one `dns1` has to be defined. Set DNS without `ip`, `mask` and `gw` does not affect the configuration (dns server will be provided by DHCP). It is not required to set DNS servers.
+* `ssl_fingerprint` is not required if `ssl` is enabled.
```

Comment:
Is this sentence correct? If 'ssl' is disabled, where is 'ssl_fingerprint' used? I don't get it... 😕

Reply:
Okay, that sentence is weird. It is a result of editing and not reading again. Will fix this.

Reply:
`ssl_fingerprint` is required if `ssl` is enabled. :+1:

Reply:
Oh, I missed to fix that sentence. And ssl_fingerprint is optional.

Reply:
Oh, so even if 'ssl' is enabled, 'ssl_fingerprint' is not required? Then the sentence is correct, but we should rewrite it to make it cleanerer. Could you please commit a change? Thanks for your work @kleini!

Reply:
Fixed. I hope the text is more clear now.